src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
```

Comment:
Consider having 'Task' class as abstract instead since it will not be initialised.